chart/values.schema.json

```
@@ -0,0 +1,1250 @@
+                            "type": "string"
+                        },
+                        "delete_worker_pods": {
+                            "type": "string"
+                        },
+                        "run_as_user": {
+                            "type": "string"
+                        },
+                        "fs_group": {
+                            "type": "string"
+                        },
+                        "git_dags_folder_mount_point": {
+                            "type": "string"
+                        },
+                        "dags_volume_mount_point": {
+                            "type": "string"
+                        },
+                        "dags_volume_claim": {
+                            "type": "string"
+                        },
+                        "dags_volume_subpath": {
+                            "type": "string"
+                        },
+                        "git_repo": {
+                            "type": "string"
+                        },
+                        "git_branch": {
+                            "type": "string"
+                        },
+                        "git_sync_rev": {
+                            "type": "string"
+                        },
+                        "git_sync_depth": {
+                            "type": "string"
+                        },
+                        "git_sync_root": {
+                            "type": "string"
+                        },
+                        "git_sync_dest": {
+                            "type": "string"
+                        },
+                        "git_sync_container_repository": {
+                            "type": "string"
+                        },
+                        "git_sync_container_tag": {
+                            "type": "string"
+                        },
+                        "git_sync_init_container_name": {
+                            "type": "string"
+                        },
+                        "git_sync_run_as_user": {
+                            "type": "string"
+                        },
+                        "git_ssh_known_hosts_configmap_name": {
+                            "type": "string"
+                        },
+                        "git_ssh_key_secret_name": {
+                            "type": "string"
+                        },
+                        "git_sync_credentials_secret": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "kubernetes_secrets": {
+                    "type": "object",
+                    "properties": {
+                        "AIRFLOW__CORE__SQL_ALCHEMY_CONN": {
+                            "type": "string"
+                        },
+                        "AIRFLOW__CORE__FERNET_KEY": {
+                            "type": "string"
+                        }
+                    }
```

It should be a free-form object. We don't want to limit the name of the keys.
```suggestion
                    "additionalProperties": { "type": "string" }
```

---

Indeed. Change done.